package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
```

Comment:
package.json and package.json shouldn't be in the main directory.  what is this stuff about